<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationWindow</class>
 <widget class="QDialog" name="OperationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(11, 58, 151);</string>
  </property>
  <widget class="QGroupBox" name="boxOp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>411</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(246, 255, 255);</string>
   </property>
   <property name="title">
    <string>Operations</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(1, 174, 238);
color: rgb(246, 255, 255);
font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Get Cash</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;background-color: rgb(1, 174, 238);
color: rgb(246, 255, 255);</string>
      </property>
      <property name="text">
       <string>Send Cash</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_6">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;background-color: rgb(1, 174, 238);
color: rgb(246, 255, 255);</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;background-color: rgb(1, 174, 238);
color: rgb(246, 255, 255);</string>
      </property>
      <property name="text">
       <string>History</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>521</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(246, 255, 255);</string>
   </property>
   <property name="title">
    <string>Profile</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QLabel" name="lBalance">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;color:rgb(144, 255, 103)</string>
      </property>
      <property name="text">
       <string>[balance]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;color: rgb(246, 255, 255);</string>
      </property>
      <property name="text">
       <string>Welcome:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;color: rgb(246, 255, 255);</string>
      </property>
      <property name="text">
       <string>Your balance:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lName">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;color: rgb(246, 255, 255);</string>
      </property>
      <property name="text">
       <string>[full name]</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;color: rgb(246, 255, 255);</string>
      </property>
      <property name="text">
       <string>credit limit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="lCredit">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;color: rgb(246, 255, 255);</string>
      </property>
      <property name="text">
       <string>[credit]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
